File: src/routes/executionsDate.js
const routering = require('express').Router();
const { validateToken } = require("../jwt/jeisonWebToken");
const currentHostRunsExecutions = require('../controllers/executionsDate')

routering.get('/executionsDate',validateToken,(req,res)=>{
    console.log("Entro por executions date desde endPoint")
    currentHostRunsExecutions.executionDate(req,res);
})

module.exports = routering

File: src/routes/executionsDate.py
from flask import Blueprint, request, jsonify
from controllers.executionsDate import executionDate

routerExecutionsDate = Blueprint('routering', __name__)

@routerExecutionsDate.route('/executionsDate', methods=['GET'])
def executions_date_route():
    print("Entro por executions date desde endPoint")
    return executionDate(request, jsonify)
